pylibftdi/driver.py

```diff
@@ -153,6 +153,7 @@ def fdll(self):
             # most args/return types are fine with the implicit
             # int/void* which ctypes uses, but some need setting here
             self._fdll.ftdi_get_error_string.restype = c_char_p
+            self._fdll.ftdi_get_library_version.restype = ftdi_version_info
```

Comment:
This will need to be gated by a check to support libftdi < 1.0; it may make more sense to move it down below.